Assets/DemonicWaveAreaScript.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DemonicWaveAreaScript : MonoBehaviour {
+
+    public SkillSettings skillSettings;
+
+    private List<GameObject> hitTargets = new List<GameObject>();
+
+	void Start () {
+        Destroy(this.gameObject, skillSettings.duration);
+	}
+
+    void OnTriggerStay(Collider other)
+    {
+        //if the hitTarget is not the ai return
+        if (other.transform.tag != Constants.ENEMY_TAG)
+            return;
+        if (!hitTargets.Contains(other.gameObject)) {
+           //todo: after maarten's info add damage system (dot/instant)
+             hitTargets.Add(other.gameObject);
+
+            //TODO: Dummyscript is just for demo so the if can be removed later on
```

Comment:
Remove todo if not valid and directly assign to AIManager instead of component and casting if no dummyscirpt is needed

Reply:
removed the todo of dummy script, the other todo is valid